.travis.yml

```diff
@@ -2,12 +2,19 @@ language: python
 sudo: false
 python:
   - "2.7"
+  - "3.4"
+  - "3.5"
 
 env:
   - DJANGO="Django==1.8.15"
   - DJANGO="Django==1.9.10"
   - DJANGO="Django==1.10.1"
 
+matrix:
+  include:
+    - python: "3.3"
+      env: DJANGO="Django==1.8.15"
```

Comment:
Why? I don't it's wrong, I just don't understand this. 
Also, if it works in 3.4 and 3.5 on 1.8, 1.9 and 1.10 why bother also testing this combination?